15/umbraco-cms/customizing/development-flow/vite-package-setup.md

```diff
@@ -63,12 +63,14 @@ npm install
 npm install -D @umbraco-cms/backoffice
 ```
 
-{% hint style="info" %}
-Optionally you can use `--legacy-peer-deps` in the installation command to avoid installing Umbraco´s sub-dependencies like TinyMCE and Monaco Editor:\
-`npm install --legacy-peer-deps -D @umbraco-cms/backoffice`
+ * Ensure that you install the version relevant to your Umbraco installation. You can find the appropriate version on the [@umbraco-cms/backoffice npm page](https://www.npmjs.com/package/@umbraco-cms/backoffice). Alternatively, you can manually update the version in your `package.json` file to match your setup.
```

Comment:
I think generally we should try to add warnings/notes related to checking specific versions, **before** the steps to install.
Otherwise, people might install the wrong thing, only to read this note afterwards 😅 

Also, the bullets are quite long and contain a lot of info. Consider if it makes sense to just add the info to a paragraph.

Reply:
Makes sense. I've tried to incorporate your review comments. Also, I made some slight changes to other steps to ensure proper flow.